tennis-scheduler-web-app/src/components/TennisCourtInfo.jsx

```diff
@@ -1,15 +1,51 @@
-function TennisCourtInfo({ image, name, surfaceType, description }) {
+import "../styles/courts.css";
+import { useState } from "react";
+import AddTimeslot from "./AddTimeslot";
+import "../styles/courts.css";
+import axios from "axios";
```

Comment:
Try to keep imports organized e.g.  css imports one beneath the other etc. Put imports from 'react' first.